src/config/Config.ts

```diff
@@ -170,4 +170,13 @@ export class Config {
    * @type {number}
    */
   gasFeeMultiplier: number
+
+  veAllocate?: string
```

Comment:
should all work out. Just as usual the question why this is all marked as optional?

Reply:
Yes, because it's not on all networks. It's just on mainnet, goerli , mumbai